runtime/Go/antlr/atn_config.go

```diff
@@ -47,6 +48,7 @@ type BaseATNConfig struct {
 	reachesIntoOuterContext    int
 }
 
+// NewBaseATNConfig7 creates a new instance of BaseATNConfig.
```

Comment:
It's unclear to me what the "7" means. Same for 6, 5, 4, etc. below.

Reply:
Yeah, I'll be renaming them according to what they actually do to create the ATN configs